jaxrs/src/test/scala/com/cerner/beadledom/jaxrs/PATCHSpec.scala

```diff
@@ -0,0 +1,77 @@
+package com.cerner.beadledom.jaxrs
+
+import com.cerner.beadledom.jaxrs.provider.{FakeModel, FakeResource}
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.jaxrs.json.JacksonJsonProvider
+
+import org.jboss.resteasy.core.Dispatcher
+import org.jboss.resteasy.mock.{MockDispatcherFactory, MockHttpRequest, MockHttpResponse}
+import org.jboss.resteasy.plugins.server.resourcefactory.POJOResourceFactory
+import org.scalatest._
+import org.scalatest.mock.MockitoSugar
+
+import java.io._
+
+import scala.collection.JavaConverters._
+import javax.ws.rs.core.MediaType
+
+/**
+  * Spec tests for {@link PATCH}.
+  *
+  * @author Eric Christensen
+  */
+class PATCHSpec extends FunSpec with BeforeAndAfterAll with ShouldMatchers with MockitoSugar {
```

Comment:
I'd like to see a test with an actual stood up service. Might fit in https://github.com/cerner/beadledom/blob/c2125c7228f9a778b56a07615f00a4d729a86270/client/beadledom-client-test/src/test/scala/com/cerner/beadledom/client/ClientServiceSpec.scala